content/post/2020-10-06-survey-wave-4.Rmd

```diff
@@ -0,0 +1,264 @@
+
+First, a simple question: What percentage of respondents say that they wear a
+mask most or all of the time when they're in public?
+
+```{r mask-wearing, message=FALSE}
+library(covidcast)
+library(ggplot2)
+
+plot_label <- labs(caption = "Data from Delphi COVIDcast, covidcast.cmu.edu")
+date_label <- labs(subtitle = format(as.Date(end_day), "%B %e, %Y"))
+
+mask <- covidcast_signal("fb-survey", "wip_smoothed_wwearing_mask",
+                         start_day = end_day, end_day = end_day,
+                         geo_type = "state")
+plot(mask, range = c(55, 100),
+     title = "% wearing masks in public most or all the time") +
+    plot_label + date_label
+```
+
+Because the survey runs continuously, collecting data on mask-wearing every day,
+we will be able to track how the prevalence of mask-wearing changes over time.
+This may help public health officials target public awareness campaigns, and
+also aid researchers trying to understand where the virus is most likely to
+spread and what measures are most effective in preventing it.
+
+### Testing
+
+Access to COVID testing is important for many reasons: readily available tests
+make it possible for cases to be detected sooner, before the virus can spread,
+and help contact tracers do their work of tracking down new cases. Shortages of
+tests can impede contact tracing and can also be harmful in settings like
+nursing homes, where many residents are particularly vulnerable to COVID-19 and
+staff must be tested regularly to ensure residents aren't exposed.
+
+Testing data is published by state health authorities and aggregated by
+organizations such as the [COVID Tracking Project](https://covidtracking.com/);
+our own [COVIDcast map](https://covidcast.cmu.edu/) also includes data from
+antigen tests produced by Quidel, a major test product manufacturer. But a
+survey allows us to ask people directly whether they have been tested, and
+allows us to understand *who* has been tested.
+
+For example, we can ask: What percentage of people in each state say they were
+tested in the last 14 days?
+
+```{r pct-tested, message=FALSE}
+tested <- covidcast_signal("fb-survey", "wip_smoothed_wtested_14d",
+                           start_day = end_day, end_day = end_day,
+                           geo_type = "state")
+plot(tested, range = c(6, 25),
+     title = "% tested in the last 14 days") +
+    plot_label + date_label
+```
+
+We can see a strong variation across the country---in North Dakota, roughly a
+quarter of people were tested in the last 14 days!
+
+It's worth thinking about this result in more detail. If we take North Dakota's
+adult population, 25% of that population is roughly twice as many tests as North
+Dakota's publicly reported testing data says were conducted during that time
+period. But North Dakota's published testing figures [only include PCR tests,
+not COVID antigen
+tests](https://khn.org/news/lack-of-antigen-test-reporting-leaves-country-blind-to-the-pandemic/),
```

Comment:
Since the KHN article mentions North Dakota specifically, what I'll do instead here is rework the paragraph above a bit. Where I previously mentioned understanding *who* has been tested, which as you point out we don't elaborate on at all, I'll instead link to the Atlantic article and give some motivation using that.